src/main/java/com/gotocompany/firehose/config/SerializerConfig.java

```diff
@@ -0,0 +1,14 @@
+package com.gotocompany.firehose.config;
+
+import com.gotocompany.firehose.config.converter.SerializerConfigConverter;
+import org.aeonbits.owner.Config;
+
+import java.util.Map;
+import java.util.function.Function;
+
+public interface SerializerConfig extends Config {
```

Comment:
"There is no need for this separate interface. Can you move this configuration to HttpSinkConfig?"